.circleci/config.yml

```diff
@@ -1,12 +1,12 @@
 version: 2.1
 
 references:
-  circleci_docker_primary: &circleci_docker_primary trussworks/circleci-docker-primary:822fac1c30f3bb7d5d595bed5d2dc86265c4f2f0
+  circleci: &circleci trussworks/circleci:6986bb9022e5a83599feb66a7128a2d0fa12732a
```

Comment:
I think you forgot a digit or two in the hash.

Reply:
hmm. I think its good I triple checked and there was no changes to the hash

Reply:
alright. circleci is pissed about something and i assumed.

Reply:
its mad that I havent did the TF13 stuff

Reply:
totally worked when I updated everything to TF13